src/pages/office/office.html

```diff
@@ -274,71 +276,129 @@ <h4>Test Summary</h4>
+          <ion-col>
+            <ion-row class="spacing-row"></ion-row>
+            <ion-row>
+              <driving-fault-comment [index]="$i" [competency]="drivingFault.propertyName"
+                [faultCount]="drivingFault.count" [faultName]="drivingFault.name"
+                [invalidIndicator]="isCtrlDirtyAndInvalid('drivingFaultCtrl' + $i)"
+                [parentForm]="form">
+              </driving-fault-comment>
+            </ion-row>
+            <ion-row class="validation-message-row" align-items-center>
+              <div class="validation-text" [class.ng-invalid]="isCtrlDirtyAndInvalid('drivingFaultCtrl' + $i)">
+                Provide a comment
+              </div>
+            </ion-row>
           </ion-col>
+
         </ion-row>
-      </ion-grid>
-    </ion-card-content>
-  </ion-card>
+      </ion-card-content>
+    </ion-card>
 
-  <ion-card id="dangerous-fault" *ngIf="(pageState.dangerousFaults$ | async).length > 0">
-    <ion-card-header>
-      <h4>Dangerous defects</h4>
-    </ion-card-header>
-    <ion-card-content>
-      <ion-row class="mes-component-row mes-row-separator" id="dangerous-commentary-label"
-        *ngFor="let dangerousFault of (pageState.dangerousFaults$ | async)">
-        <ion-col class="component-label" align-self-center col-1>
-          <dangerous-fault-badge [showBadge]="true"></dangerous-fault-badge>
-        </ion-col>
-        <ion-col class="component-label" col-29 align-self-center>
-          <label class="fault-label">{{dangerousFault}}</label>
-        </ion-col>
-        <ion-col>
-          <ion-row class="spacing-row"></ion-row>
-          <ion-row>
-            <textarea #dangerousFaultComment id="dangerous-commentary-text" row="5" class="mes-data"
-              [value]="pageState.dangerousFaultComments$ | async">
-            </textarea>
-          </ion-row>
-          <ion-row class="spacing-row"></ion-row>
-        </ion-col>
-      </ion-row>
-    </ion-card-content>
-  </ion-card>
+    <ion-card id="driving-fault-no-commentary"
+      *ngIf="!(pageState.displayDrivingFaultComments$ | async) && (pageState.drivingFaultCount$ | async) > 0">
+      <ion-card-header>
+        <h4>Driving faults</h4>
+      </ion-card-header>
+      <ion-card-content>
+        <ion-row class="driving-fault-no-commentary-row" id="driving-fault-commentary-label"
+          *ngFor="let drivingFault of (pageState.drivingFaults$ | async)">
+          <ion-col class="component-label" align-self-center col-1>
+            <div class="counter-icon">
+              <driving-faults-badge class="counter driving-faults" [count]="drivingFault.count"></driving-faults-badge>
+            </div>
+          </ion-col>
+          <ion-col class="component-label" col-58 align-self-center>
+            <label class="fault-label">{{drivingFault.name}}</label>
+          </ion-col>
+        </ion-row>
+      </ion-card-content>
+    </ion-card>
+  </form>
```

Comment:
This file is 410 lines. Can you please split it into multiple components?

Reply:
I have been working on breaking this component up into a component per row, this change will come soon. 👍

Reply:
That would be outside of the scope of this ticket. Ross has made a component for the driving faults. John is currently working on adding components for other areas.

Reply:
Okay, nice one!